jgrapht-core/src/main/java/org/jgrapht/alg/interfaces/MatchingAlgorithm.java

```diff
@@ -17,24 +17,63 @@
  */
 package org.jgrapht.alg.interfaces;
 
-import java.util.*;
+import java.util.Set;
 
 /**
- * Allows to derive <a href="http://en.wikipedia.org/wiki/Matching_(graph_theory)">matching</a> from
- * given graph
+ * Allows to derive a <a href="http://en.wikipedia.org/wiki/Matching_(graph_theory)">matching</a> of
+ * a given graph.
  *
  * @param <V> the graph vertex type
  * @param <E> the graph edge type
- * 
  */
 public interface MatchingAlgorithm<V, E>
 {
     /**
+     * Default tolerance used by algorithms comparing floating point values.
+     */
+    double DEFAULT_EPSILON = 1e-9;
```

Comment:
Hm, good idea to add the DEFAULT_EPSILON here. We should probably do this for the flow/cut implementations as well.